legal-api/src/legal_api/services/__init__.py

```diff
@@ -39,12 +39,12 @@
 from .naics import NaicsService
 from .namex import NameXService
 from .pdf_service import PdfService
-from .queue import QueueService
+# from .queue import QueueService
 from .warnings.business import check_business
 from .warnings.warning import check_warnings
 
 flags = Flags()  # pylint: disable=invalid-name; shared variables are lower case by Flask convention.
-queue = QueueService()  # pylint: disable=invalid-name; shared variables are lower case by Flask convention.
+# queue = QueueService()  # pylint: disable=invalid-name; shared variables are lower case by Flask convention.
```

Comment:
this was re-enabled when updates for CI were made for legal api. 

Disabled so legal api works for now.

Will need to revisit when we are closer to doing integration testing.